.github/ISSUE_TEMPLATE.md

```diff
@@ -5,10 +5,10 @@ Number of the clients:
 Cluster size, i.e. the number of Hazelcast cluster members:
 OS version : Windows/Linux/OSX
 
-#### Expected behaviour
+#### Expected behavior
 
 
-#### Actual behaviour
+#### Actual behavior
 
 
-#### Steps to reproduce the behaviour
\ No newline at end of file
+#### Steps to reproduce the behavior
```

Comment:
`behaviour` is not wrong, it's written like that in British English.

Reply:
My bad, but Google itself uses as `behavior`,
https://support.google.com/analytics/answer/2785577?hl=en#zippy=%2Cin-this-article
I think we should choose the same phrase with them as being a Go client.
Also, overall usage of `behavior` is bigger.
<img width="862" alt="Screen Shot 2022-08-01 at 10 07 02" src="https://user-images.githubusercontent.com/43863606/182094105-542cd659-23c5-48b9-aa7c-c15875cf4d6f.png">
Wdyt ?

Reply:
Google is a US company, so that makes sense for them :) I don't agree with your reasoning, but `behavior` is OK.